assignment/impl/src/java/org/sakaiproject/assignment/impl/AssignmentServiceImpl.java

```diff
@@ -4908,15 +4908,15 @@ private Optional<Long> createCategoryForGbAssignmentIfNecessary(
 
         if (!StringUtils.isBlank(categoryName)) {
             List<CategoryDefinition> toCategoryDefinitions
-                = gradebookService.getCategoryDefinitions(toGradebookId);
+                = gradebookService.getCategoryDefinitions(toGradebookId, toGradebookId);
```

Comment:
Same here, I don't know if we should leave this as is and call the method with two arguments when there's only one in the caller.